setup.py

```diff
@@ -56,8 +56,8 @@ def read_requirements(filename):
     "smdebug_rulesconfig==1.0.1",
     "importlib-metadata>=1.4.0,<7.0",
     "packaging>=20.0",
-    "pandas",
```

Comment:
Can we try this approach where we can provide documentation for minimal installation like:
```
pip install --no-deps sagemaker
pip install -r minimal_requirements.txt // this has the minimal requirements without pandas.
```
we can put this in a shell script and call it lambda-install.sh or something and put it in sdk,

Reply:
ok. i'm on it

Reply:
There is no easy way to provide the `minimal_requirements.txt` file to users, since installing the SDK with pip only copies the Python files to the install path, not the requirement files.

Reply:
Do you want to include requirement files in the Pypi package?

Reply:
is there a way to do it? the only way to solve the issue in a backwards compatible way is to provide documentation to users for AWS Lambda support with no-deps instructions above.

Reply:
if its not possible to include files in pip dist, lets just provide adequate documentation to users.

Reply:
Yes, it's possible

Reply:
The requirement files are not stored in the source folder of the SDK, so shipping them with the SDK is not recommended. I went with the documentation route.

Reply:
https://sagemaker--4222.org.readthedocs.build/en/4222/installation.html